update_duktape.sh

```diff
@@ -0,0 +1,14 @@
+SRC_LOCATION=duktape/src/main/jni/
```

Comment:
How about a shebang up here? We don't know what shell a user is running which might break even the most simple things like star expanding.

```
#!/bin/bash
```